comp.html

```diff
@@ -0,0 +1,968 @@
+                    style="width: 517px; height: 140px; border:0; transform: scale(1); overflow:hidden;margin:2rem 0;border-radius:1rem;"
+                    sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                        
+                    <h3 class="secondary-txt-showcase">@Text only cards</h3>
+                    <div class="column">
+                        <div class="card" >
+                            <div class="card-info" >
+                                <h3>Card Title</h3>  
+                                <small>Card Subtitle</small>          
+                                <p class="card-info-para">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Rem, illo dolores voluptatum molestiae voluptate suscipit debitis nam qui beatae tenetur.</p>
+                                <button class="btn primary">Show More</button>
+                            </div>
+                        </div>
+                    </div>
+
+                    <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=panda-syntax&wt=sharp&l=css&ds=false&dsyoff=20px&dsblur=68px&wc=false&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=15px&lh=128%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522column%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card%2522%2520%253E%250A%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522card-info%2522%2520%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch3%253ECard%2520Title%253C%252Fh3%253E%2520%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Csmall%253ECard%2520Subtitle%253C%252Fsmall%253E%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%2520class%253D%2522card-info-para%2522%253EDescription%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520primary%2522%253EShow%2520More%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+                    style="width: 544px; height: 231px; border:0; transform: scale(1); overflow:hidden;margin:2rem;border-radius:1rem;"
+                    sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+
+                    <h3 class="secondary-txt-showcase">@Horizontal Cards</h3>
+                    <div class="column">
+                        <div class="card-horztl">
```

Comment:
Optional: a clearer name for class can be used for user friendliness as card-horizontal or card-h only